gradle/libs.versions.toml

```diff
@@ -4,6 +4,7 @@ coreTesting = "2.2.0"
 kotlin = "2.0.0"
 coreKtx = "1.13.1"
 junit = "5.10.2"
+junitVersion = "1.2.1"
```

Comment:
You have `junitTestingExt` already. Why create another one with an ambiguous name?